ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -429,8 +429,10 @@ objects:
         fi
         if [[ "${CLUSTER_TYPE}" = "aws" ]]; then
             master_type=null
+            worker_type=5.4xlarge
```

Comment:
did you mean m5?

Reply:
also this is the default, so i don't think you should be setting this here.

Reply:
updated the logic.  This value was the hard coded value from below so it should remain the same should it not?